app/controllers/org_admin/templates_controller.rb

```diff
@@ -53,6 +53,7 @@ def create
       @template = Template.new(params[:template])
       @template.org_id = current_user.org.id
       @template.description = params['template-desc']
+      @template.links = (params["template-links"].present? ? JSON.parse(params["template-links"]) : {"funder": [], "sample_plan": []})
```

Comment:
rather than using here the ternary, if you use like that:
```
 @template.links = JSON.parse(params["template-links"]) if params["template-links"].present?
```
will only update template.links with whatever comes from the client otherwise will default to whatever is defined in JUST one place. Anyway, when you create a new Template it should take the defaults from the db unless they have not been removed in a newer commit.